features/works/chapter_edit.feature

```diff
@@ -274,45 +274,51 @@ Feature: Edit chapters
       And I go to the works page
     Then "First work" should appear before "A Whole New Work"
 
-
-  Scenario: Posting a new chapter with a co-creator does not add them to
-  previous or subsequent chapters
+  Scenario: Posting a new chapter with a co-creator does not add them to previous or subsequent chapters
 
     Given I am logged in as "karma" with password "the1nonly"
+      And the user "sabrina" allows co-creators
       And I post the work "Summer Friends"
     When a chapter is set up for "Summer Friends"
-    Then I should not see "Chapter co-creators"
-    When I add the co-author "sabrina"
+      And I invite the co-author "sabrina"
       And I post the chapter
+    Then I should not see "sabrina"
+    When the user "sabrina" accepts all co-creator invites
+      And I view the work "Summer Friends"
     Then I should see "karma, sabrina"
-    When I follow "Previous Chapter"
-    Then I should see "Chapter by karma"
+      And I should see "Chapter by karma"
+    When I follow "Next Chapter"
+    Then I should not see "Chapter by"
     When a chapter is set up for "Summer Friends"
-    Then I should see "Chapter co-creators"
+    Then I should see "Current co-creators"
       And the "sabrina" checkbox should not be checked
     When I post the chapter
     Then I should see "Chapter by karma"
 
-
-  Scenario: You should be able to edit a chapter to add a co-creator who is not
-  already on the work
+  Scenario: You can edit a pre-existing chapter to invite a new co-creator
 
     Given I am logged in as "karma" with password "the1nonly"
+      And the user "amy" allows co-creators
       And I post the work "Forever Friends"
       And a chapter is added to "Forever Friends"
     When I view the work "Forever Friends"
       And I view the 2nd chapter
       And I follow "Edit Chapter"
-    Then I should not see "Chapter co-creators"
-    When I add the co-author "amy"
+      And I invite the co-author "amy"
       And I post the chapter
-    Then I should see "amy, karma"
+    Then I should not see "amy, karma"
       And 1 email should be delivered to "amy"
-      And the email should contain "You have been listed as a co-creator on the following work"
+      And the email should contain "The user karma has invited your pseud amy to be listed as a co-creator on the following chapter"
       And the email should not contain "translation missing"
+    When the user "amy" accepts all co-creator invites
+      And I view the work "Forever Friends"
+    Then I should see "amy, karma"
+      And I should see "Chapter by karma"
+    When I follow "Next Chapter"
+    Then I should not see "Chapter by"
 
 
-  Scenario: You should be able to edit a chapter to add a co-creator who is
+  Scenario: You can edit a chapter to add (not invite) a co-creator who is
```

Comment:
This sends a creatorship_notification email now, right? We should probably check for it in this test, if so.